sigs.yaml

```diff
@@ -1547,6 +1547,13 @@ sigs:
     frequency: weekly
     url: https://zoom.us/j/289465529
     archive_url: https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit
+  - description: Network Policy API Meeting
+    day: Monday
+    time: "13:00"
+    tz: PT (Pacific Time)
+    frequency: weekly
+    url: TBD
```

Comment:
I think one of the SIG leads has to setup a reoccuring zoom call and send me a link?